src/extension.ts

```diff
@@ -28,43 +33,70 @@ export class DaCeVSCode {
         context.subscriptions.push(SdfgViewerProvider.register(context));
 
+        // Register the SDFG outline view.
+        context.subscriptions.push(OutlineProvider.register(context));
+
         // Register necessary commands.
-        vscode.commands.registerCommand('transformationView.refreshEntry',
+        context.subscriptions.push(vscode.commands.registerCommand(
+            'transformationView.refreshEntry',
             () => {
                 transformationsProvider.refresh();
             }
-        );
-        vscode.commands.registerCommand('sdfg.applyTransformation', (t) =>
-            daceInterface.applyTransformation(t));
-        vscode.commands.registerCommand('sdfg.previewTransformation', (t) =>
-            daceInterface.previewTransformation(t));
-        vscode.commands.registerCommand('sdfg.previewHistoryPoint', (h) =>
-            daceInterface.previewHistoryPoint(h));
-        vscode.commands.registerCommand('sdfg.applyHistoryPoint', (h) =>
-            daceInterface.applyHistoryPoint(h));
-        vscode.commands.registerCommand('dace.openOptimizerInTerminal', () =>
-            daceInterface.startDaemonInTerminal());
-        vscode.commands.registerCommand('dace.installDace', () => {
-            const term = vscode.window.createTerminal('Install DaCe');
-            term.show();
-            term.sendText('pip install git+https://github.com/spcl/dace.git');
-        });
+        ));
+        context.subscriptions.push(vscode.commands.registerCommand(
+            'sdfgOutline.refreshEntry',
+            () => {
+                DaCeVSCode.getInstance().activeEditorSendPost({
+                    type: 'refresh_outline',
+                });
+            }
+        ));
+        context.subscriptions.push(vscode.commands.registerCommand(
+            'sdfg.applyTransformation',
+            (t) => daceInterface.applyTransformation(t)
+        ));
+        context.subscriptions.push(vscode.commands.registerCommand(
+            'sdfg.previewTransformation',
+            (t) => daceInterface.previewTransformation(t)
+        ));
+        context.subscriptions.push(vscode.commands.registerCommand(
+            'sdfg.previewHistoryPoint',
+            (h) => daceInterface.previewHistoryPoint(h)
+        ));
+        context.subscriptions.push(vscode.commands.registerCommand(
+            'sdfg.applyHistoryPoint',
+            (h) => daceInterface.applyHistoryPoint(h)
+        ));
+        context.subscriptions.push(vscode.commands.registerCommand(
+            'dace.openOptimizerInTerminal',
+            () => daceInterface.startDaemonInTerminal()
+        ));
+        context.subscriptions.push(vscode.commands.registerCommand(
+            'dace.installDace',
+            () => {
+                const term = vscode.window.createTerminal('Install DaCe');
+                term.show();
+                term.sendText(
+                    'pip install git+https://github.com/spcl/dace.git'
```

Comment:
now it can just be `pip install dace`. New version released

Reply:
Good point, changed!